### ISSUE TITLE
Access to Login Page Blocked by CORS Policy

### ISSUE THREAD

**Issue by rleath82** [dotnet/AspNetCore.Docs#19119]
Hello, I've tried this tutorial many times and I keep getting the same error when I try to navigate to the login page. The error I'm getting is:

![error](https://user-images.githubusercontent.com/38876176/86674923-2ea23180-bfc7-11ea-966c-9b9749f5a5d7.PNG)

Any idea what I can do to fix this?

Thanks for your time,

Bob

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 520b9821-ecef-20af-a601-ea36c4297d88
* Version Independent ID: da9c8ac9-4d7a-4b40-56a2-a4bfe7083567
* Content: [Secure an ASP.NET Core Blazor WebAssembly hosted app with Azure Active Directory B2C](https://docs.microsoft.com/en-us/aspnet/core/blazor/security/webassembly/hosted-with-azure-active-directory-b2c?view=aspnetcore-3.1)
* Content Source: [aspnetcore/blazor/security/webassembly/hosted-with-azure-active-directory-b2c.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/blazor/security/webassembly/hosted-with-azure-active-directory-b2c.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
Your `Authority` doesn't look right in your *Client* app. Looks like you have the wrong tenant domain in there (`findastage` and `findastageb2c` seem to be in conflict). Try ...

```
https://findastage.b2clogin.com/findastage.onmicrosoft.com/b2c_1_signupsignin
```

In the *Server* app, your `Instance` should probably be `https://findastage.b2clogin.com/`, and the tenant `Domain` should probably be `findastage.onmicrosoft.com`. Either that, or `findastageb2c` should appear in both spots where you have `findastage`. Again, they usually match.

**Comment by rleath82** [comment]
This still isn't working, unfortunately. I've tried making those changes and still the same result. My repo is https://github.com/rleath82/FAS if you want to take a look. I can also email you my AAD configs.

Thanks again

**Comment by guardrex** [comment]
That doesn't look like the correct scope URI host. Try ...

```csharp
options.ProviderOptions.DefaultAccessTokenScopes.Add("https://findastage.onmicrosoft.com/408fbb37-87fc-433c-a94a-9e5dc121d4f7/API.Access");
```

https://docs.microsoft.com/en-us/aspnet/core/blazor/security/webassembly/hosted-with-azure-active-directory-b2c?view=aspnetcore-3.1#access-token-scopes

Be sure to use an incognito/in private browser run **_for every single run of the app_** no matter how small the change to the app or to the settings in the Azure portal. Lingering cookies from prior runs can break the setup as you test and throw false errors. For information on how to configure VS to do that automatically, see the [Troubleshoot section of the topic](https://docs.microsoft.com/en-us/aspnet/core/blazor/security/webassembly/hosted-with-azure-active-directory-b2c?view=aspnetcore-3.1#troubleshoot).

If it still won't run, check your Azure AD B2C configuration again against the topic's guidance. I think configuration is the source of your app's trouble (and/or perhaps stale cookies in the browser).

We're a small team that just works on docs. We don't have the staffing to provide general programming support. If you discover something specifically wrong with a topic, please do open a new issue from the bottom of the topic. For general support, ask on a support channel, such as ...

* [Stack Overflow](https://stackoverflow.com/questions/tagged/blazor)
* [ASP.NET Core Slack](http://tattoocoder.com/aspnet-slack-sign-up/)
* [ASP.NET Gitter](https://gitter.im/aspnet/Blazor)
